src/runtime_src/core/tools/xbmgmt2/flash/xmc.cpp

```diff
@@ -556,3 +556,16 @@ bool XMC_Flasher::hasSC()
     } catch (...) {}
     return sc_presence;
 }
+
+bool XMC_Flasher::fixedSC()
+{
+    bool is_sc_fixed = false;
+    std::string errmsg;
```

Comment:
Is this string needed?

Reply:
No it's not. Good catch. I'll fix it with the upcoming cleanup :)